In [13]:
import pandas as pd
import numpy as np
buyer = pd.read_csv('BuyerRatio.csv')

In [14]:
buyer

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [15]:
buyer.drop(["Observed Values"], axis = 1, inplace = True)

In [16]:
buyer

East  West  North  South
0    50   142    131     70
1   435  1523   1356    750

In [17]:
import scipy.stats as stats
from scipy.stats import chi2_contingency

In [18]:
p_value = stats.chi2_contingency(buyer)

In [19]:
p_value

(1.595945538661058,
 0.6603094907091882,
 3,
 array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
        [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))

In [20]:
p_value[1]

0.6603094907091882

# p-value (0.66)> α (0.05), hence we can't reject the null hypothesis. Proportion of male and female across regions is same.